docs/source/api.rst

```diff
@@ -11,6 +11,17 @@ Browse the Open API documentation
 - https://public.vulnerablecode.io/api/schema/ for the OpenAPI schema
 
 
+How to use OpenAPI documentation
+--------------------------------------
+
+The API documentation is available at https://public.vulnerablecode.io/api/docs/.
+To use the endpoints you need to authenticate with an API key. Request your API key
+from https://public.vulnerablecode.io/account/request_api_key/. Once you have
+your API key, click on the ``Authorize`` button on the top right of the page and enter
+your API key in the ``value`` field with ``Token`` prefix, so if your token is "1234567890abcdef"
+then you have to enter this: ``Token 1234567890abcdef``.
+
+
```

Comment:
The lines below "Enable the API key authentication" and "Create an API key-only user" are irrelevant for end users as they do not have access to this. These should be moved to a new section related to API usage administration for on-premise deployments, either at the bottom, or a completely new and different page